controllers/index.js

```diff
@@ -16,4 +14,42 @@ const getRoom = (roomId, callback) => {
   });
 };
 
-module.exports = { getRoom };
+// Method adds document
+const postRoom = (dataItem, callback) => {
+  db.Room.create(dataItem, (err, room) => {
+    if (err) {
```

Comment:
Consider using the promise syntax .then() .catch(), which mongoose methods tend to be compatible with. It would clean up the code quite a bit.